File: CarDealerWebAPI/Core.CarDealer/DTO/MessageDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Core.CarDealer.DTO
{
    public class MessageDTO
    {
        [Required]
        public Guid SenderId { get; set; }
        [Required]
        public string Subject { get; set; }
        [Required]
        public string Content { get; set; }
        [Required]

        public Guid ReceiverId { get; set; }
    }
}

File: CarDealerWebAPI/CarDealer.Client/ClientApp/src/app/DTOs/MessageDTO.ts




export class MessageDTO
{
    private _senderId!: string;
    public get senderId(): string {
        return this._senderId;
    }
    public set senderId(value: string) {
        this._senderId = value;
    }
    private _subject!: string;
    public get subject(): string {
        return this._subject;
    }
    public set subject(value: string) {
        this._subject = value;
    }
    private _content!: string;
    public get content(): string {
        return this._content;
    }
    public set content(value: string) {
        this._content = value;
    }
    private _receiverId!: string;
    public get receiverId(): string {
        return this._receiverId;
    }
    public set receiverId(value: string) {
        this._receiverId = value;
    }
}